pkg/common/ipsec/utils.go

```diff
@@ -4,6 +4,9 @@
 package ipsec
```

Comment:
The commit title is too long and that's why checkpatch is complaining. It should also be a title instead of a sentence and it's good practice to prepend it with the part of the code base that's affected (ex. `cmd:`, `bpf:`). Run `git log` in the directory you're making changes in to see examples.

Reply:
Gotcha! Is it better now?
Thanks!

Reply:
It's better but "bug" isn't a code base area :wink:  You could also be a bit more specific in what is fixed. For example, `cmd: Handle non-AEAD IPsec keys in encrypt status`.

Reply:
Fixed.